docs/authentication.md

```diff
@@ -0,0 +1,70 @@
+# Topics
+* [Auth configuration](#auth-configuration)
+  * [OIDC Authentication](#oidc-Authentication)
+  * [Basic authentication](#basic-authentication)
+
+## Auth configuration
+
+FastTrackML supports 2 types of authentication: OIDC based authentication and Basic authentication.
+
+### OIDC Authentication
+
+To enable OIDC authentication, FastTrackML should be run with next command line parameters:
+```
+--auth-oidc-client-id client_id --auth-oidc-client-secret client_secret --auth-oidc-provider-endpoint http://127.0.0.1 --auth-oidc-admin-role admin --auth-oidc-claim-roles groups --auth-oidc-scopes email openid 
```

Comment:
maybe break line with backslash-newline after each option so it's easier to read?